src/window/gui/Gui.cpp

```diff
@@ -473,6 +473,7 @@ void Gui::DrawMenu() {
     if (!ImGui::DockBuilderGetNode(dockId)) {
         ImGui::DockBuilderRemoveNode(dockId);
         ImGui::DockBuilderAddNode(dockId, ImGuiDockNodeFlags_NoTabBar);
+        ImGui::DockBuilderSetNodeSize(dockId, ImVec2(viewport->Size.x, viewport->Size.y));
```

Comment:
Without this, dock sizing may not be consistent/accurate. I've added it in here to prevent future issues.